src/features/layout/Modal.module.scss

```diff
@@ -0,0 +1,45 @@
+@import "../../common/colors";
+
+dialog.modal {
+  $padding: 1em;
+  $border: 1px solid use-color("base-light");
+  border: $border;
+  display: flex;
+  flex-flow: column nowrap;
+  overflow: visible;
+  padding: $padding;
+
+  .header {
+    align-items: start;
+    display: flex;
+    flex-direction: row nowrap;
+    flex-shrink: 1;
+    justify-content: space-between;
+    width: 100%;
+
+    .subtitle {
+      color: use-color("base-light");
+      font-weight: normal;
+    }
+  }
+
+  .footer {
+    border-top: $border;
+    display: flex;
+    flex-direction: row;
+    gap: 0.333em;
+    justify-content: end;
```

Comment:
See also the other comment, I think the linter wants you to use `flex-end` here.